packages/api-explorer/src/components/DocSdkUsage/DocSdkUsage.tsx

```diff
@@ -40,31 +48,59 @@ interface DocSdkUsageProps {
  *  links to the source files
  */
 export const DocSdkUsage: FC<DocSdkUsageProps> = ({ method }) => {
-  const tabs = useTabs()
   const { examples } = useContext(LodeContext)
   const languages = findExampleLanguages(examples, method.name)
   if (languages.length === 0) return <></>
 
+  const tableExamples = prepareExampleTableData(
+    languages,
+    examples,
+    method.operationId
+  )
```

Comment:
This should be done using DataTable because of all the things we get from DataTable "for free." Replicating DataTable's functionality for sorting, pagination, filtering, and searching is not reasonable. The layout portion of the examples table is the most trivial part.